c/tskit/tables.c

```diff
@@ -5600,9 +5601,9 @@ tsk_table_collection_check_integrity(tsk_table_collection_t *self, tsk_flags_t o
 {
     int ret = TSK_ERR_GENERIC;
     tsk_size_t j;
-    double left, right, position;
+    double time, left, right, position;
     double L = self->sequence_length;
-    double *time = self->nodes.time;
+    double *times = self->nodes.time;
```

Comment:
``node_time`` here would be more consistent with other usage I think (but not a strong opinion, there's no right answer here)

Reply:
It seems (from below, line 6008) that you want to keep `time` (singular) for this array of times?